process-automation/latest/model/forms.md

```diff
@@ -468,3 +469,100 @@ The actions that can be run against a form are:
 | Validate | Run validation against the form. Any errors can be seen in the log history at the bottom of the Modeling Application and are flagged in a pop-up box. |
 | Save | Save any changes made to the form. |
 | Delete | Delete the form. |
+
+## Form rules
+
+Form rules can be used to populate one field depending on the response given by a user in another field. In the example described here you create a form rule called **how_to_address** that has two fields on it, the first is called **Gender** and the second is called **Title**. The first field is a dropdown list and contains the options **Man**, **Woman**, and **Other**. If **Woman** is selected then the second field is automatically populated with **Ms**. The form rule ensures that any interaction with the form will contain consistent responses. This is a simple example and your form rules could be more involved.
+
+These instructions are in two parts. You first create a form, and then second create the form rules.
+
+> **Note:** Form rules are specific to your installation and configuration.
+
+### Create the form
+
+First you must create a form that can be used to configure the form rule:
+
+1. Sign into the Modeling Application and open a Project, click the three dots next to **Forms** and then select the **+** icon.
```

Comment:
I shall leave this the way it is because it is a bigger job that just this one process. The whole docs need to be looked at and updated. I believe alot of the steps will need to be updated and not just this one. 
This work was also done as part of 7.3 and not 7.4.
I will add this work to my prio list as it is something I have seen already, thanks.
Cheers Pablo.